src/event/handlers/LogMemberLeaveHandler.ts

```diff
@@ -0,0 +1,27 @@
+import { Constants, GuildMember, MessageEmbed, TextChannel } from "discord.js";
+import EventHandler from "../../abstracts/EventHandler";
+import { EMBED_COLOURS, MEMBER_ROLE, LOG_CHANNEL_ID } from "../../config.json";
+import DateUtils from "../../utils/DateUtils";
+
+class LogMemberLeaveHandler extends EventHandler {
+	constructor() {
+		super(Constants.Events.GUILD_MEMBER_REMOVE);
+	}
+
+	async handle(guildMember: GuildMember): Promise <void> {
+		const embed = new MessageEmbed();
+
+		embed.setTitle(`${guildMember.user.tag} left the server.`);
```

Comment:
For consistency with the other logs, can the title please be "Member Left" with the description of embed being "User: `@mention`" (you should still be able to mention users who are no longer in the guild).